driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -523,7 +522,9 @@ pub mod tests {
             run_solver: |_, _, _| Ok(()),
             read_output: |_| Err(std::io::Error::last_os_error()),
             fee: Some(fee),
-            solver_type: SolverType::StandardSolver,
+            solver_type: SolverConfig::StandardSolver {
+                solver_time_limit: 180,
```

Comment:
is this a default or did we just hardcode the three minutes in right here?

Reply:
here, it is just a test parameter, which is not relevant for the actual test